python/paddle/fluid/tests/unittests/ir/inference/test_conv_act_onednn_fuse_pass.py

```diff
@@ -160,34 +165,44 @@ def sample_program_config(self, draw):
         if draw(st.booleans()):
             conv_bias_shape = [f_shape[0]]
             inputs = {
-                "Input": ["input_x"],
-                "Filter": ["filter"],
-                "ResidualData": ["residualdata"],
-                "Bias": ["conv_bias"],
+                'Input': ['input_x'],
+                'Filter': ['filter'],
+                'ResidualData': ['residualdata'],
+                'Bias': ['conv_bias'],
             }
             weights = {
-                "filter": TensorConfig(shape=f_shape),
-                "conv_bias": TensorConfig(shape=conv_bias_shape),
+                'filter': TensorConfig(shape=f_shape),
+                'conv_bias': TensorConfig(shape=conv_bias_shape),
             }
             use_mkldnn = True
         else:
             inputs = {
-                "Input": ["input_x"],
-                "Filter": ["filter"],
-                "ResidualData": ["residualdata"],
+                'Input': ['input_x'],
+                'Filter': ['filter'],
+                'ResidualData': ['residualdata'],
             }
-            weights = {"filter": TensorConfig(shape=f_shape)}
+            weights = {'filter': TensorConfig(shape=f_shape)}
             use_mkldnn = False
 
         # 11. Generate legal act type of conv2d
         act_type = draw(
-            st.sampled_from(["relu", "leaky_relu", "relu6", "swish"])
+            st.sampled_from(
+                [
+                    'relu',
+                    'leaky_relu',
+                    'relu6',
+                    'swish',
+                    'mish',
+                    'hard_sigmoid',
+                    'hard_swish',
```

Comment:
This list is not complete. Please add all activations from _onednn_reuse.h_ `GetSupportedActivations()` 
```suggestion
                    'abs',
                    'clip',
                    'gelu',
                    'hard_sigmoid',
                    'hard_swish',
                    'leaky_relu',
                    'mish',
                    'relu',
                    'relu6',
                    'sigmoid',
                    'sqrt',
                    'swish',
                    'tanh'
```